.github/workflows/cd.yml

```diff
@@ -0,0 +1,128 @@
+name: CD
+
+on:
+  release:
+    types: [published]
+  schedule:
+    - cron: '0 0 * * 0' # Every Sunday at midnight UTC
+  workflow_dispatch:
+
+env:
+  FORCE_COLOR: 3
+
+permissions: {}
+
+jobs:
+  build-python-distributions:
+    name: Build sdist and wheel
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+        with:
+          persist-credentials: false
+
+      - name: Setup Python
+        uses: actions/setup-python@a26af69be951a213d495a4c3e4e4022e16d87065 # v5
+        with:
+          python-version: '3.11'
+
+      - name: Install jlpm
+        run: pip install "jupyterlab>=4.0.0,<5"
+
+      - name: Build the extension artifacts
+        run: pipx run build --output-dir distributions
+
+      - name: Upload the extension artifacts
+        uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02 # v4
+        with:
+          name: sdist-and-wheel
+          path: distributions/jupytereverywhere*
+          if-no-files-found: error
+
+  build-jupyterlite-app:
+    name: Build JupyterLite deployment
+    needs: build-python-distributions
```

Comment:
`build-jupyterlite-app` gets the JupyterEverywhere wheel built in `build-python-distributions` above for further usage.